main.css

```diff
@@ -0,0 +1,770 @@
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  grid-column: 3;
+  grid-row: 2;
+}
+
+.gridded {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 15px;
+  grid-template-areas: "head head head head";
+  padding: 2em 6em;
+  width: 100%;
+}
+
+.grid-address {
+  grid-area: head;
+}
+
+.grid-address a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.gridded-image {
+  width: 100%;
+  height: 150px;
+}
+
+.news a {
+  text-decoration: none;
+}
+
+.funds-address a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.fund-head a {
+  color: white;
+  text-decoration: none;
+  margin-left: 5em;
+}
+
+.fund-footer a,
+.footer-footer a {
+  display: block;
+  padding: 0.5em 1em;
+  font-size: 0.85em;
+  text-decoration: none;
+  color: #444;
+}
+
+.common-text a {
+  text-decoration: none;
+  color: white;
+  font-size: 1.2em;
+}
+
+.list-grid-mid a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.list-grid-footer a {
+  color: #999;
+  text-decoration: none;
+  font-size: 0.9em;
+}
+
+.social-icons a {
+  text-decoration: none;
+  border-radius: 20px;
+  border: 1px solid #999;
+  background-color: #666;
+  padding: 10px 12px;
+  color: white;
+}
+
+.social-list a {
+  color: #777;
+  text-decoration: none;
+  padding: 0 0.5em;
+  font-size: 0.9em;
+}
+
+.news h4 a {
+  color: black;
+}
+
+.news p,
+.news p a {
+  color: #999;
+  font-size: 0.9em;
+}
+
+.fund-grid {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-areas: "head head head head";
+  padding: 2em 6em;
+  grid-gap: 20px;
+  width: 100%;
+}
+
+.funds-address {
+  grid-area: head;
+  display: flex;
+  justify-content: space-between;
+  margin: 2em 0;
+}
+
+.funds-address ul {
+  display: flex;
+  list-style: none;
+}
+
+.funds-address li {
+  padding: 0 0.5em;
+  border-right: 1px solid #f42;
+}
+
+.fund-head {
+  background-color: #f42;
+  padding: 0.5em 0;
+}
+
+.fund-grid img {
+  width: 60px;
+  height: 60px;
+  position: relative;
+  margin: -1.5em 0.4em 0 0.4em;
+}
+
+.fund-body {
+  background-color: #ffebe8;
+  padding: 1em 0;
+  margin-top: -2.5em;
+}
+
+.fund-body h5,
+.fund-body h6 {
+  text-align: center;
+  font-weight: bold;
+  font-size: 0.85em;
+}
+
+.fund-footer {
+  margin-top: 0.5em;
+  border-bottom: 0.1px solid #f42;
+}
+
+.footer-footer {
+  margin-top: 0.5em;
+}
+
+.list-grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  padding: 2em 6em;
+  grid-gap: 20px;
+}
+
+.list-grid h2 a {
+  color: #f42;
+  text-decoration: none;
+}
+
+.apps-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/apps.png);
+  background-position: center center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.gear-body {
+  background: linear-gradient(rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.8)), url(./TNW-Images/gear.png);
+  background-position: center center;
+  background-size: cover;
```

Comment:
[Required] Even though the stylings of elements are good. there are a lot of repeated codes that are not necessary. Let's follow the DRY principle and optimize the CSS file. [This article](https://speckyboy.com/good-bad-css-practices/) might be helpful.